src/eda.py

```diff
@@ -86,6 +30,51 @@ def create_and_save_chart(df, col, save_path):
     train_df = (
         pd.read_csv(train_path, low_memory=False).set_index("index").rename_axis(None)
     )
+    
+    # Converts values of P_SEX column into numeric
+    sex = {'M': 1, 'F': 0, 'missing': 'missing'}
+    train_df['P_SEX'] = [sex[item] for item in train_df['P_SEX']]
+    
+    # Creates the list of features to feed into distribution plots
+    features = list(set(train_df.columns.values) - set(['index', 'FATALITY']))
+
+    # Creates and saves distribution plots when Fatality = False
+    Chart_False = (
+        alt.Chart(train_df)
+        .mark_bar(opacity=0.5)
+        .encode(
+            x=alt.X(alt.repeat("row"), type="quantitative"),
+            y=alt.Y("count()", title="Number of collisions"),
+            color=alt.Color("FATALITY", scale=alt.Scale(range=["blue"]), legend=None))
+        .properties(width=150, height=150)
+        .repeat(
+            row=features,
+            columns=1,
+            title="FATALITY = False")
+        .resolve_scale(y="independent")
+        .transform_filter(alt.FieldOneOfPredicate(field="FATALITY", oneOf=[False]))
+    )
+    
+    Chart_False.save(f"{save_path}/Distribution_of_no_fatality.png")
```

Comment:
Get rid of slash after save path to keep consistent with other scripts and README

Reply:
done